### PR TITLE
Fix the examples of Az.Storage Syntax Errors

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

Fix the examples of Az.Storage Syntax Errors

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage.Management/help/Add-AzRmStorageContainerLegalHold.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ This command adds legal hold tags to a Storage blob container with Storage accou
 
 ### Example 3: Add legal hold tags to all Storage blob containers in a Storage account with pipeline
 ```
-PS C:\>Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" | Add-AzRmStorageContainerLegalHold -Tag  tag1,tag2,tag3
+PS C:\>Get-AzStorageContainer -Name "myContainer" | Add-AzRmStorageContainerLegalHold -Tag  tag1,tag2,tag3
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Add-AzRmStorageContainerLegalHold.md:58]: The fix is wrong. The correct fix is use cmdlet name "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Close-AzStorageFileHandle.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ This command closes all file handles on a file.
 
 ### Example 2: Close all file handles which is opened 1 day ago on a file directory
 ```
-PS C:\> Get-AzStorageFileHandle -ShareName "mysharename" -Path 'dir1/dir2' -Recursive | ? {$_.OpenTime.DateTime.AddDays(1) -lt (Get-Date)} | Close-AzStorageFileHandle -ShareName "mysharename"
+PS C:\> Get-AzStorageFileHandle -ShareName "mysharename" -Path 'dir1/dir2' -Recursive | Where-Object {$_.OpenTime.DateTime.AddDays(1) -lt (Get-Date)} | Close-AzStorageFileHandle -ShareName "mysharename"
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Close-AzStorageFileHandle.md:77]: The before script also work with "?", please indicate why need change to "Where-Object"

File: src/Storage/Storage.Management/help/Get-AzRmStorageContainerImmutabilityPolicy.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ This command gets ImmutabilityPolicy of a Storage blob containers with Storage a
 
 ### Example 3: Get ImmutabilityPolicy of a Storage blob container with Storage container object
 ```
-PS C:\>$containerObject = Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" -Name "myContainer"
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Get-AzRmStorageContainerImmutabilityPolicy.md:55]: The change should be change cmdlet name to "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Lock-AzRmStorageContainerImmutabilityPolicy.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ This command locks ImmutabilityPolicy of a Storage blob container, with Storage
 
 ### Example 3: Lock ImmutabilityPolicyof a Storage blob container, with container object
 ```
-PS C:\>$containerObject = Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" -Name "myContainer"
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Lock-AzRmStorageContainerImmutabilityPolicy.md:64]: The fix should be change cmdlet name to "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Remove-AzRmStorageContainer.md [hunk 5]
```diff
@@ -52,7 +52,7 @@ This command removes a Storage blob container with Storage account object and co
 
 ### Example 3: Remove all Storage blob containers in a Storage account with pipeline
 ```
-PS C:\>Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" | Remove-AzRmStorageContainer -Force
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Remove-AzRmStorageContainer.md:55]: The fix should be change cmdlet name to "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Remove-AzRmStorageContainerImmutabilityPolicy.md [hunk 6]
```diff
@@ -61,7 +61,7 @@ This command removes unlocked ImmutabilityPolicy of a Storage blob container, wi
 
 ### Example 3: Remove unlocked ImmutabilityPolicy of a Storage blob container, with container object
 ```
-PS C:\>$containerObject = Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" -Name "myContainer"
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Remove-AzRmStorageContainerImmutabilityPolicy.md:64]: The fix should be change cmdlet name to "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Remove-AzRmStorageContainerLegalHold.md [hunk 7]
```diff
@@ -53,7 +53,7 @@ This command removes legal hold tags from a Storage blob container with Storage
 
 ### Example 3: Remove legal hold tags from all Storage blob containers in a Storage account with pipeline
 ```
-PS C:\>Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" | Remove-AzRmStorageContainerLegalHold -Tag  tag1
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Remove-AzRmStorageContainerLegalHold.md:56]: The fix should be change cmdlet name to "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Remove-AzRmStorageShare.md [hunk 8]
```diff
@@ -72,7 +72,7 @@ This command removes a Storage file share with Storage account object and share
 
 ### Example 3: Remove all Storage file shares in a Storage account with pipeline
 ```
-PS C:\>Get-AzStorageShare -ResourceGroupName "myResourceGroup" -StorageAccountName "myStorageAccount" | Remove-AzRmStorageShare -Force
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Remove-AzRmStorageShare.md:75]: The fix should be change cmdlet name to "Get-AzRmStorageShare"

File: src/Storage/Storage.Management/help/Set-AzRmStorageContainerImmutabilityPolicy.md [hunk 9]
```diff
@@ -90,7 +90,7 @@ This command extend ImmutabilityPolicy of a Storage blob container, with Storage
 
 ### Example 3: Update ImmutabilityPolicy of a Storage blob container
 ```
-PS C:\>$containerObject = Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" -Name "myContainer"
+PS C:\>$containerObject = Get-AzStorageContainer -Name "myContainer"
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Set-AzRmStorageContainerImmutabilityPolicy.md:93]: The fix should be change cmdlet name to "Get-AzRmStorageContainer"

File: src/Storage/Storage.Management/help/Update-AzRmStorageContainer.md [hunk 10]
```diff
@@ -54,7 +54,7 @@ This command disables public access on a Storage blob container with Storage acc
 
 ### Example 3: Set public access as Blob for all Storage blob containers in a Storage account with pipeline
 ```
-PS C:\>Get-AzStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" | Update-AzRmStorageContainer -PublicAccess Blob
+PS C:\>Get-AzStorageContainer -Name "myContainer" | Update-AzRmStorageContainer -PublicAccess Blob
```
Comment (blueww) [inline src/Storage/Storage.Management/help/Update-AzRmStorageContainer.md:57]: The fix should be change cmdlet name to "Get-AzRmStorageContainer"

**Review by blueww — CHANGES_REQUESTED** [review]
Thanks for the PR!
Please help to fix my comments.

**Comment by CaptainFanZzz** [comment]
> Thanks for the PR! Please help to fix my comments.

Thanks for your help! 
In order to make the doc formal, we may avoid using alias.

**Review by blueww — DISMISSED** [review]
LGTM
Thanks for the fix!

**Review by ziyuezh576 — DISMISSED** [review]

File: src/Storage/Storage.Management/help/Close-AzStorageFileHandle.md (hunk 2 above)
Comment (isra-fel) [reply src/Storage/Storage.Management/help/Close-AzStorageFileHandle.md:77]: ? is the alias for Where-Object

**Review by isra-fel — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**